.github/workflows/prerelease.yml

```diff
@@ -16,21 +16,6 @@ env:
   TAG: ${{ github.event.release.tag_name }}
 
 jobs:
-  snyk:
-    name: Run security checks via snyk
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-      - name: Login to DockerHub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
-          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
-      - name: Scan code for vulnerabilities
-        env:
-          SNYK_TOKEN: ${{ secrets.COREINT_SNYK_TOKEN }}
-        run: make ci/snyk-test
-
```

Comment:
Are we removing Snyk from the pipelines? Is it that now we run it differently?

Reply:
yes since this is running with the SAAS version of snyk